app/controllers/staff/application_controller.rb

```diff
@@ -13,6 +13,14 @@ def require_staff
     end
   end
 
+  def require_contact_email
+    if @event.contact_email.empty?
+      session[:target] = request.path
```

Comment:
Usually after an event is updated the user is redirected to the event show page. I felt that was annoying because in this instance the user was trying to send teammate invites. This way, in this circumstance, after they go and add the contact email for the event they are redirected back to the teammates page so they can continue sending invites.